agent/engine/service_connect/manager_linux.go

```diff
@@ -15,17 +15,90 @@ package serviceconnect
 
 import (
 	"fmt"
+	"os"
+	"path/filepath"
 
 	apicontainer "github.com/aws/amazon-ecs-agent/agent/api/container"
 	apitask "github.com/aws/amazon-ecs-agent/agent/api/task"
 	dockercontainer "github.com/docker/docker/api/types/container"
 )
 
+const (
+	defaultRelayPathContainer  = "/var/run/ecs/relay/"
+	defaultRelayPathHost       = "/var/run/ecs/instance/relay/"
+	defaultRelayFileName       = "envoy_xds.sock"
+	defaultRelayENV            = "APPMESH_XDS_ENDPOINT"
```

Comment:
does this have to be `APPMESH`? or we can change to `SERVICE_CONNECT_XDS_ENDPOINT`?

Reply:
this is what the current Agent container looks for. If we change the container to look for SC or APPNET prefixes we can make a corresponding change here at that time.